src/main/java/duke/controller/Parser.java

```diff
@@ -79,12 +79,12 @@ public void handleTodo(String input, TaskList taskList) throws DukeException {
     /**
      * Adds either a Deadline Task or Event Task based on the command.
      *
-     * @param command Command.
-     * @param input User input.
+     * @param command  Command.
+     * @param input    User input.
      * @param taskList List of Tasks.
      * @throws DukeException If the user input is incorrect.
      */
-    public void handleTasksWithTime(String command, String input, TaskList taskList) throws DukeException {
+    public String handleTasksWithTime(String command, String input, TaskList taskList) throws DukeException {
```

Comment:
Perhaps you can shorten the line to make your code more readable according to coding guidelines?